src/promptbase/utils/utils.py

```diff
@@ -15,7 +15,7 @@
         "url": os.getenv("AZURE_OPENAI_EMBEDDINGS_URL"),
     },
     "azure": {
-        "headers": {"Authorization": f"Bearer {os.getenv('AZURE_OPENAI_CHAT_API_KEY')}"},
+        "headers": {"api-key": f"{os.getenv('AZURE_OPENAI_CHAT_API_KEY')}"},
```

Comment:
Do we document anywhere exactly which values should go into these environment variables? I'm guessing that this is the key which users can get from the AOAI playground if they select 'View Code' ?

Reply:
Yes, that's the right key. It's also visible from a keys panel in the portal. We will document necessary environment variables in the readme.